src/software/jetson_nano/services/motor.h

```diff
@@ -36,6 +36,8 @@ class MotorService
     TbotsProto::MotorStatus poll(const TbotsProto::MotorControl& motor_control,
                                  double time_elapsed_since_last_poll_s);
 
+    void disableVelocity(double current_velocity, double previous_velocity);
```

Comment:
missing docs

Reply:
Thanks for the catch, I'll probably actually be removing this function since it's not being used in my robot auto test